packages/vue/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@lingui/vue",
```

Comment:
Add the 

```
"publishConfig": {
  "access": "public"
},
```

to allow the package to be published by CI first-time

Reply:
done